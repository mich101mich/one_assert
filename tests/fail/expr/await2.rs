fn main() {
    let fut = async { true };
    one_assert::assert!(fut.await);
}
