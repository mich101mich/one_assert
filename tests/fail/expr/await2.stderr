error[E0728]: `await` is only allowed inside `async` functions and blocks
 --> tests/fail/expr/await2.rs:3:29
  |
1 | fn main() {
  | --------- this is not `async`
2 |     let fut = async { true };
3 |     one_assert::assert!(fut.await);
  |                             ^^^^^ only allowed inside `async` functions and blocks
